apps/obsidian/scripts/build.ts

```diff
@@ -0,0 +1,27 @@
+import { compile } from "./compile";
```

Comment:
will generalize all of the `scripts/` to work for all apps (as best we can) later

Reply:
https://roamresearch.com/#/app/discourse-graphs/page/PJX-ytImj